java-client/src/main/java/org/opensearch/client/opensearch/_types/mapping/FieldType.java

```diff
@@ -111,6 +111,8 @@ public enum FieldType implements JsonEnum {
 
     Shape("shape"),
```

Comment:
Should we remove `Shape` altogether since it is not supported by OS?

Reply:
I was unsure, but I can remove it

Reply:
It is unusable, right? (as per issue) So we don't need to keep it, thank you.